cognite/client/data_classes/relationships.py

```diff
@@ -156,7 +156,66 @@ def dump(self, camel_case: bool = False):
 
 
 class RelationshipUpdate(CogniteUpdate):
-    pass
+    """Update applied to a single relationship
+
+    Args:
+        id (int): A server-generated ID for the object.
```

Comment:
maybe?
```suggestion
```

Reply:
Good catch. I specifically added a custom init for RelationshipUpdate so that it could not be instantiated with `id`